setup.py

```diff
@@ -50,7 +49,7 @@ def get_readme() -> str:
         "docker",
         "matplotlib",
         "numpy",
-        "pandas",
+        "pandas[html]",
```

Comment:
Needed to use `pandas.read_html`. See https://pandas.pydata.org/pandas-docs/stable/getting_started/install.html#html